job/src/main/java/no/nav/common/job/leader_election/ShedLockLeaderElectionClient.java

```diff
@@ -37,21 +48,24 @@
 
     public ShedLockLeaderElectionClient(LockProvider lockProvider) {
         this.lockProvider = lockProvider;
+        scheduler = Executors.newSingleThreadScheduledExecutor();
+        hostname = EnvironmentUtils.resolveHostName();
+
+        scheduler.scheduleWithFixedDelay(this::keepLockAlive, 0, CHECK_LOCK_EVERY_SECONDS, TimeUnit.SECONDS);
```

Comment:
Flisespikkeri dette også, men syntes kanskje CHECK_LOCK_INTERVAL eller CHECK_LOCK_INTERVAL_IN_SECONDS eller noe sånt hadde vært en tanke mer beskrivende for hva konstanten betyr